### ISSUE TITLE
`hoarder bookmarks list` seems to be limited to 20 entries

### ISSUE THREAD

**Issue by skirmess** [MohamedBassem/hoarder-app#135]
The `hoarder` cli only lists 20 bookmarks. If I would have to guess I would assume this is because of pagination.

```
$ hoarder bookmarks list | grep ' id: ' | wc -l
      20
```

**Comment by MohamedBassem** [comment]
Oh, good call. Should be an easy fix. Will fix it over the weekend :) Thanks for the report!

**Comment by MohamedBassem** [comment]
I pushed CLI version `0.13.0` to address this bug :) Thanks again for the report.

**Comment by skirmess** [comment]
I've updated the npm version to 0.13.2 (or I think I did, I have this in my package-lock.json)

```
    "node_modules/@hoarderapp/cli": {
      "version": "0.13.2",
```

Now it seems to be limited to 100

```
$ hoarder bookmarks list | grep id: | wc -l
     100
```

**Comment by MohamedBassem** [comment]
Do you by any chance have exactly 100 bookmarks? :D

**Comment by skirmess** [comment]
Unfortunately, no.

Let's create 10 more:
```
$ for i in 1 2 3 4 5 6 7 8 9 10
> do
>
> hoarder lists add-bookmark --list mrci6l1iszhqntw1r07pjjc6 --bookmark $(hoarder bookmarks add --link https://example.com/test$i | sed -n -e "s/.*id: '//p" | sed -e "s/',//")
> done
Successfully added bookmark from list
Successfully added bookmark from list
Successfully added bookmark from list
Successfully added bookmark from list
Successfully added bookmark from list
Successfully added bookmark from list
Successfully added bookmark from list
Successfully added bookmark from list
Successfully added bookmark from list
Successfully added bookmark from list
```

Total is still 100:

```
$ hoarder bookmarks list | grep id: | wc -l
      100
```

And when I check the new list I don't see any bookmarks.

```
$ hoarder bookmarks list --list-id mrci6l1iszhqntw1r07pjjc6
[]
```

With the web interface I can see the 10 links I've created above under  `https://my-server/dashboard/lists/mrci6l1iszhqntw1r07pjjc6`

![image](https://github.com/MohamedBassem/hoarder-app/assets/8441536/13f0c268-5533-4bdb-8de2-d8573a280c09)

**Comment by MohamedBassem** [comment]
Interesting ... Let me look into why this is happening. Thanks for taking the time to test it out.

**Comment by MohamedBassem** [comment]
Turned out, `console.log` has a default array limit of 100 when printing an object :)

I also found a bug in the way I was doing pagination resulting in some bookmarks created in the same second to be skipped. I'll need to fix that but that will require a server release. Tracking that issue in #140

**Comment by MohamedBassem** [comment]
The fix should be released in the CLI version `0.13.3`.

**Comment by skirmess** [comment]
Both issues fixed. Thank you!

```
$ hoarder bookmarks list | grep id: | wc -l
    3892
$ hoarder bookmarks list --list-id mrci6l1iszhqntw1r07pjjc6 | grep id: | wc -l
      10
```
